comp/forwarder/eventplatform/eventplatformimpl/epforwarder.go

```diff
@@ -152,18 +152,18 @@ var passthroughPipelineDescs = []passthroughPipelineDesc{
 		//   by aggregator.
 		defaultInputChanSize: 10000,
 	},
-	{
-		eventType:                     eventplatform.EventTypeNetworkPath,
-		category:                      "Network Path",
-		contentType:                   logshttp.JSONContentType,
-		endpointsConfigPrefix:         "network_path.forwarder.",
-		hostnameEndpointPrefix:        "netpath-intake.",
-		intakeTrackType:               "netpath",
-		defaultBatchMaxConcurrentSend: 10,
-		defaultBatchMaxContentSize:    pkgconfig.DefaultBatchMaxContentSize,
-		defaultBatchMaxSize:           pkgconfig.DefaultBatchMaxSize,
-		defaultInputChanSize:          pkgconfig.DefaultInputChanSize,
-	},
+	// {
+	// 	eventType:                     eventplatform.EventTypeNetworkPath,
+	// 	category:                      "Network Path",
+	// 	contentType:                   logshttp.JSONContentType,
+	// 	endpointsConfigPrefix:         "network_path.forwarder.",
+	// 	hostnameEndpointPrefix:        "netpath-intake.",
+	// 	intakeTrackType:               "netpath",
+	// 	defaultBatchMaxConcurrentSend: 10,
+	// 	defaultBatchMaxContentSize:    pkgconfig.DefaultBatchMaxContentSize,
+	// 	defaultBatchMaxSize:           pkgconfig.DefaultBatchMaxSize,
+	// 	defaultInputChanSize:          pkgconfig.DefaultInputChanSize,
+	// },
```

Comment:
Please remove the code instead of commenting out. We can always re-introduce it or revert the removal at a later date.

Reply:
same comment here (from NPM)

Reply:
Done 👍